sdk/messaging/azservicebus/internal/stress/tests/finite_send_and_receive.go

```diff
@@ -39,23 +39,20 @@ func FiniteSendAndReceiveTest(remainingArgs []string) {
 	sender, err := client.NewSender(queueName, nil)
 	sc.PanicOnError("failed to create sender", err)
 
-	const messageLimit = 500
+	const messageLimit = 50000
```

Comment:
Add comment on why the messageLimit value is 50000

Reply:
Done.